backend/src/seeds/achievements.ts

```diff
@@ -0,0 +1,35 @@
+import { type Knex } from 'knex';
+
+const Activities = {
+    CYCLING: 'cycling',
+    RUNNING: 'running',
+    WALKING: 'walking',
+};
+
+const Achievements = [
+    {
+        name: 'First 5 km Cycling',
+        activity: Activities.CYCLING,
+        requirement: '5 km',
```

Comment:
Maybe it will be better to split this into a requirement field and a requirement_metric field represented as an enum of available requirements metrics? This way, we wouldn't need to parse it all the time.

Reply:
Thanks for checking) I will fix everything according to the comments.